src/components/general/Popup.js

```diff
@@ -132,7 +132,7 @@ const Popup = ({
             break
           case 'bottomRight':
             top = rect.y
-            left = rect.x
+            left = rect.x + triggerRect.width
```

Comment:
i *think* this is probably good for the other use case, but to confirm with @sddioulde

Reply:
yeah sg. fyi this was feedback I got from Julian about the rewards use case - not sure that it applies to profile but I suspect it does.

Reply:
Hmm. Depends on what bottom right really means for us. Wouldn't be the end of the world for sort use case, but let's confirm

Reply:
resolved on slack